src/pybamm/solvers/idaklu_jax.py

```diff
@@ -8,19 +8,14 @@
 
 from functools import lru_cache
 
-import importlib.util
-import importlib
-
 logger = logging.getLogger("pybamm.solvers.idaklu_jax")
 
-idaklu_spec = importlib.util.find_spec("pybamm.solvers.idaklu")
-if idaklu_spec is not None:
-    try:
-        idaklu = importlib.util.module_from_spec(idaklu_spec)
-        if idaklu_spec.loader:
-            idaklu_spec.loader.exec_module(idaklu)
-    except ImportError:  # pragma: no cover
-        idaklu_spec = None
+try:
+    import pybammsolvers.idaklu as idaklu
+except ImportError as e:  # pragma: no cover
+    idaklu = None
+    print(f"Error loading idaklu: {e}")
+
```

Comment:
The IDAKLU solver is optional here, but `pybammsolvers` also needs to be an optional dependency for this to work correctly and be guarded under `pybamm[idaklu]` or a similar name (maybe `[solvers]` because the plan is to add CVODES too?)

Reply:
I had this in here for testing and error handling. I think there is probably other cleanup that can be done as well.